src/Mod/MeshPart/App/CMakeLists.txt

```
@@ -93,70 +93,23 @@ INSTALL(TARGETS MeshPart DESTINATION ${CMAKE_INSTALL_LIBDIR})
 
 
 ################################ flat mesh ###############################
-if (BUILD_FLAT_MESH)
-    if (FREECAD_USE_PYBIND11)
-        SET(FLATMESH_SRCS
-            MeshFlattening.cpp
-            MeshFlattening.h
-            MeshFlatteningNurbs.h
-            MeshFlatteningNurbs.cpp
-            MeshFlatteningLscmRelax.h
-            MeshFlatteningLscmRelax.cpp
-            MeshFlatteningPy.cpp
-        )
-
-
-        add_library(flatmesh SHARED ${FLATMESH_SRCS})
-        SET_PYTHON_PREFIX_SUFFIX(flatmesh)
-        target_link_libraries(flatmesh ${PYTHON_LIBRARIES} ${MeshPart_LIBS})
-
-        SET_BIN_DIR(flatmesh flatmesh /Mod/MeshPart)
-        install(TARGETS flatmesh DESTINATION ${CMAKE_INSTALL_LIBDIR})
-    else()
-        if(NOT FREECAD_LIBPACK_USE OR FREECAD_LIBPACK_CHECKFILE_CLBUNDLER OR FREECAD_LIBPACK_CHECKFILE_VERSION)
-            # boost-python >= 1.67 on some platforms has suffix
-            if (FORCE_BOOST_PY_SUFFIX)
-                set(BOOST_PY_SUFFIX ${FORCE_BOOST_PY_SUFFIX})
-            else ()
-                set(BOOST_PY_SUFFIX ${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR})
-            endif ()
-
-            find_package( Boost COMPONENTS python${BOOST_PY_SUFFIX} )
-            if (NOT Boost_PYTHON${BOOST_PY_SUFFIX}_FOUND)
-                # try just the major version
-                find_package( Boost COMPONENTS python${PYTHON_VERSION_MAJOR} )
-                if (NOT Boost_PYTHON${PYTHON_VERSION_MAJOR}_FOUND)
-                    # unversioned
-                    find_package( Boost COMPONENTS python REQUIRED)
-                endif()
-            endif()
-
-            if(Boost_FOUND)
-                include_directories(${Boost_INCLUDE_DIRS})
-                SET(FLATMESH_SRCS
-                    MeshFlattening.cpp
-                    MeshFlattening.h
-                    MeshFlatteningNurbs.h
-                    MeshFlatteningNurbs.cpp
-                    MeshFlatteningLscmRelax.h
-                    MeshFlatteningLscmRelax.cpp
-                    MeshFlatteningBoostPython.cpp
-                )
-
-
-                add_library(flatmesh SHARED ${FLATMESH_SRCS})
-                SET_PYTHON_PREFIX_SUFFIX(flatmesh)
-
-                if (BUILD_DYNAMIC_LINK_PYTHON)
-                    target_link_libraries(flatmesh ${PYTHON_LIBRARIES})
-                endif(BUILD_DYNAMIC_LINK_PYTHON)
-
-                target_link_libraries(flatmesh ${MeshPart_LIBS} ${Boost_LIBRARIES})
-
-                SET_BIN_DIR(flatmesh flatmesh /Mod/MeshPart)
-                install(TARGETS flatmesh DESTINATION ${CMAKE_INSTALL_LIBDIR})
-            endif()
-        endif()
-    endif()
-endif(BUILD_FLAT_MESH)
+if (BUILD_FLAT_MESH AND FREECAD_USE_PYBIND11)
```

```suggestion
if (BUILD_FLAT_MESH)
```

seems to me like we should remove the FREECAD_USE_PYBIND11 option since there is no other alternative for this